internal/toolkit/mock_toolkit.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.4.0-beta. DO NOT EDIT.
+// Code generated by mockery v0.0.0-dev. DO NOT EDIT.
```

Comment:
hmm, not sure why this happened. Maybe just revert it

Reply:
Hi @jkutner , I guess this is happening because I installed `mockery` using `go get` rather than `brew` and that's why I have the development version of it. 
Related issue: https://github.com/vektra/mockery/issues/362

However, I tried to install `mockery` using brew and then the code generated has correct version but since it installs latest version of `mockery`, that is, `v2.9.0`, I am not sure if I should use that? Or do you suggest to just simply edit it to `v2.4.0-beta`?  Thanks

Reply:
I'm not sure. Did you run `go get` (without options) from the root of the github-actions repo?

Reply:
Nope, I installed from the instructions here: https://github.com/vektra/mockery#go-get

This was the command: `go get github.com/vektra/mockery/v2/.../` and this version mismatch is a known issue in `mockery`. Also found here: https://github.com/vektra/mockery/issues/381

Reply:
I removed the binary installed using `go get` method and re installed `mockery` using brew. It is not showing the correct version but the version is `v2.9.0` in this case.

Reply:
Most likely, this version number is set at compile time. Which is what `vetktra/mockery` is likely doing for their official releases using `brew`.

`go get ...`, which fetches and builds the tool, won't be smart enough to know what version number to set.

https://github.com/vektra/mockery/blob/master/pkg/config/config.go#L3-L4

Reply:
I guess you are right, the version is hardcoded it seems, @aemengo. However, I have fixed it somehow by locally installing `mockery` with its pre-built binary of the version `v2.4.0-beta` we are using. Here's a link to that: https://github.com/vektra/mockery/releases/tag/v2.4.0-beta